### ISSUE TITLE
Python and Tensorboard version?

### ISSUE THREAD

**Issue by CatalinOAlexandru** [Lornatang/SRGAN-PyTorch#31]
Hi, I have some incompatibility problems, would you mind sharing what python version and tensorboard version you were using while you made the latest release (few days ago)?

**Comment by Lornatang** [comment]
2.4.0 and 2.5.0

**Comment by CatalinOAlexandru** [comment]
Thank you. My problem was actually with some modification I have done to my python itself. 

Without opening another issue, can I ask you @Lornatang if you obtained your results after training for 512 epochs (P-epochs) and 128 epochs (G-epochs)? I found that training for only 100 and 25 will give me an average PSNR of aprox 25 for Set14, however, the result for the butterfly won't look as good as yours (in README). Even with the high PSNR, the results from the test set won't show a very appealing look. Most probably needs more training from my side

I'm just wondering if 512 and 128 was chosen after different tests, or just a number which should work.

**Comment by CatalinOAlexandru** [comment]
In addition, without making a pull request, in train.py [L148](https://github.com/Lornatang/SRGAN-PyTorch/blob/master/train.py#L148) and [L174](https://github.com/Lornatang/SRGAN-PyTorch/blob/master/train.py#L174), I believe based on your code that you should have 'sr.png' instead of 'hr.png'. Otherwise, the next line after these 2 will crash, as sr.png is not existent.

**Comment by Lornatang** [comment]
The SRGAN training process is unstable in the first stage. You should wait for it to be fully trained. In the second stage, take the highest SSIM Epoch as the final model.

**Comment by Lornatang** [comment]
sr.png is generated after each Epoch, it is generated by lr.png, and image evaluation is performed with hr.png

**Comment by CatalinOAlexandru** [comment]
> The SRGAN training process is unstable in the first stage. You should wait for it to be fully trained. In the second stage, take the highest SSIM Epoch as the final model.

By first stage, you mean training the psnr and second stace being the gan training, right? So for the best result "wait for it to be fully trained" you mean to wait the 512 epochs before Gan ?


> sr.png is generated after each Epoch, it is generated by lr.png, and image evaluation is performed with hr.png

Sorry, maybe I said it wrong. The path you set is wrong in L148 and L174. You are overwriting the hr.png

**Comment by Lornatang** [comment]
yeah

**Comment by CatalinOAlexandru** [comment]
> yeah

Thank you.

1 more note. In test.py you are linking to the path of LR images instead of ground truth [L121](https://github.com/Lornatang/SRGAN-PyTorch/blob/master/test.py#L121)

**Comment by Lornatang** [comment]
Thanks. It fixed.
